docs/sources/operators-guide/mimir-runbooks/_index.md

```diff
@@ -1386,6 +1386,49 @@ How to **fix** it:
 
 - Increase the per-tenant limit by using the `-distributor.ha-tracker.max-clusters` option (or `ha_max_clusters` in the runtime configuration).
 
+### err-mimir-sample-timestamp-too-old
+
+This error occurs when the ingester rejects a sample because its timestamp is too old as compared to the most recent timestamp received for the same tenant across all its time series.
+
+How it **works**:
+
+- If the incoming timestamp is more than 1 hour older than the most recent timestamp ingested for the tenant, the sample will be rejected.
+
+### err-mimir-sample-out-of-order
+
+This error occurs when the ingester rejects a sample because another sample with a more recent timestamp has already been ingested.
+
+How it **works**:
+
+- Currently, samples are not allowed to be ingested out of order for a given series.
+
+Common **causes**:
+
+- Your code has a single target that exposes the same time series multiple times, or multiple targets with identical labels.
+- System time of your Prometheus instance has been shifted backwards. If this was a mistake, fix the system time back to normal. Otherwise, wait until the system time catches up to the time it was changed, or delete all ingested samples with a timestamp later than system time.
```

Comment:
Deleting samples with a timestamp later than system time is something not really doable, neither in Prometheus or Mimir.

```suggestion
- System time of your Prometheus instance has been shifted backwards. If this was a mistake, fix the system time back to normal. Otherwise, wait until the system time catches up to the time it was changed.
```